classes/decision/types/traits/NotifyReviewers.php

```diff
@@ -57,7 +57,6 @@ protected function sendReviewersEmail(DecisionNotifyReviewer|ReviewerUnassign $m
                 if ($reviewAssignment) {
                     $reviewAssignment->setDateAcknowledged(Core::getCurrentDate());
                     $reviewAssignment->stampModified();
-                    $reviewAssignment->setUnconsidered(ReviewAssignment::REVIEW_ASSIGNMENT_NOT_UNCONSIDERED);
```

Comment:
Don't get the point why a notifying a review needed to update the review assignment consideration status . Weird part is this is my code when I worked on https://github.com/pkp/pkp-lib/issues/2890

Reply:
I think this is the same as https://github.com/pkp/pkp-lib/pull/8593/files#r1095049305.

Reply:
@asmecher I think this is still wrong  . the trait used in several decision class and only in decision class such as 
1. CancelInternalReviewRound
2. Accept
3. CancelReviewRound
4. Decline
5. RequestRevisions
6. Resubmit
and the purpose of method `sendReviewersEmail` is to sent mail to any reviewer associated with submission at the current review round about the taken decision . So don't see any reason to update the review assignment `considered` status at time of notifying . 

Also you see previous status was 
```php
public const REVIEW_ASSIGNMENT_NOT_UNCONSIDERED = 0; // Has never been unconsidered
```
So basically at the notify time it's setting the review assignment consider status as `0` which make no sense. Right now this status set through different action such as 
```
Just when a new assignment created --> 0 (REVIEW_ASSIGNMENT_NEW)
Read and Confirm First Time               --> 3 (REVIEW_ASSIGNMENT_CONSIDERED)
Revert Decision                                     --> 1 (REVIEW_ASSIGNMENT_UNCONSIDERED)
Read and Confirm Again                       --> 2 (REVIEW_ASSIGNMENT_RECONSIDERED)
```
Also at the end of few decisions , review round and associated review assignment get deleted .

Sorry if I am bike shedding here .

Reply:
Gotcha, and since that code (https://github.com/pkp/pkp-lib/issues/2890) has never been released, we can change it however we like without affecting users. What I meant is that the logic was probably to cause the review to be flagged as considered when the editor had obviously moved past it in the workflow. I'm confident that thanking the reviewer is a good case for marking the review flagged, but I don't think editor decisions are. I'd support removing the change to the considered status in this use case.